tests/retryable-reads/retryable-reads-002.phpt

```diff
@@ -26,19 +26,20 @@ class Observer implements MongoDB\Driver\Monitoring\CommandSubscriber
 }
 
 $manager = new MongoDB\Driver\Manager(URI, ['retryReads' => true]);
+$server = $manager->selectServer(new \MongoDB\Driver\ReadPreference('primary'));
```

Comment:
This looks necessary since we may run on a sharded cluster. Good place for a comment.